shared/go.mod

```diff
@@ -6,7 +6,7 @@ require (
 	github.com/CycloneDX/cyclonedx-go v0.7.2-0.20230625092137-07e2f29defc3
 	github.com/CycloneDX/cyclonedx-gomod v1.4.0
 	github.com/anchore/grype v0.65.2
-	github.com/anchore/stereoscope v0.0.0-20230727211946-d1f3d766295e
+	github.com/anchore/stereoscope v0.0.0-20230911191510-2fc2d6c2503b
```

Comment:
I think it's better to keep it to be aligned with syft and grype, DYT?
https://github.com/anchore/grype/blob/v0.65.2/go.mod#L13
https://github.com/anchore/syft/blob/v0.87.1/go.mod#L57